flexiblepricing/constants.py

```diff
@@ -0,0 +1,33 @@
+class FlexiblePriceStatus:
+    """Statuses for the FlexiblePrice model"""
+
+    APPROVED = "approved"
+    AUTO_APPROVED = "auto-approved"
+    CREATED = "created"
+    DOCS_SENT = "docs-sent"
+    PENDING_DOCS = "pending-docs"
```

Comment:
You can remove these, these are specific to the high degree of customization for DEDP that we won't be using in MITx Online.